feature const_fn has been removed

I'm getting this in rust-nightly CI

```
   Compiling field-offset v0.3.3
error[E0557]: feature has been removed
 --> /github/home/.cargo/registry/src/github.com-1ecc6299db9ec823/field-offset-0.3.3/src/lib.rs:3:53
  |
3 | #![cfg_attr(fieldoffset_assert_in_const_fn, feature(const_fn))]
  |                                                     ^^^^^^^^ feature has been removed
  |
  = note: split into finer-grained feature gates
```

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -1,6 +1,6 @@
 [package]
 name = "field-offset"
-version = "0.3.3"
+version = "0.3.4"
 authors = ["Diggory Blake <diggsey@googlemail.com>"]
 description = "Safe pointer-to-member implementation"
 repository = "https://github.com/Diggsey/rust-field-offset"

--- a/src/lib.rs
+++ b/src/lib.rs
@@ -1,6 +1,4 @@
 #![no_std]
-#![cfg_attr(fieldoffset_assert_in_const_fn, feature(const_if_match))]
-#![cfg_attr(fieldoffset_assert_in_const_fn, feature(const_fn))]
 #![cfg_attr(fieldoffset_assert_in_const_fn, feature(const_panic))]
 // Explicit lifetimes are clearer when we are working with raw pointers,
 // as the compiler will not warn us if we specify lifetime constraints

